<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>половинный угол расходимости дл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAlpha">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.110000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Длина трансфокатора</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxL">
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>300.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>l0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberL0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>коллимирующая линза</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxFKoll">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>delta Lkol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDeltaLKol">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxF1">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>120.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>l1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberL1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>delta L1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDeltaL1">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>F2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxF2">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>-30.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>l2 - положение F2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxL2">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>F3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxF3">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>120.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ktr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberKTr">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>l3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxL3">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>фокусирующая  линза</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxFFoc">
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>увеличение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>delta lFoc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberDeltaLFoc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>k1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberK1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>k2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberK2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QSlider" name="horizontalSliderL2">
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBoxFFoc</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxL3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxAlpha</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxFKoll</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxDeltaLKol</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxDeltaL1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
 </slots>
</ui>
